linuxdeploy-plugin-gtk.sh

```diff
@@ -67,6 +67,11 @@ else
     exit 1
 fi
 
+if [ -z "$LINUXDEPLOY" ]; then
+    echo -e "$0: LINUXDEPLOY environment variable is not set.\nDownload a suitable linuxdeploy AppImage, set the environment variable and re-run the plugin."
```

Comment:
Why are you using `-e`?

Reply:
`-e     enable interpretation of backslash escapes`
Because of newline character (`\n`). If you prefer, I can remove the newline or add a second `echo`.

Reply:
Ah, I overlooked that.